opentelemetry-zipkin/Cargo.toml

```diff
@@ -28,10 +28,10 @@ reqwest-rustls = ["reqwest", "reqwest/rustls-tls-native-roots"]
 [dependencies]
 async-trait = { workspace = true }
 once_cell = { workspace = true }
-opentelemetry = { version = "0.24", path = "../opentelemetry" }
-opentelemetry_sdk = { version = "0.24", path = "../opentelemetry-sdk", features = ["trace"] }
-opentelemetry-http = { version = "0.13", path = "../opentelemetry-http" }
-opentelemetry-semantic-conventions = { version = "0.16", path = "../opentelemetry-semantic-conventions" }
+opentelemetry = { version = "0.25", path = "../opentelemetry" }
+opentelemetry_sdk = { version = "0.25", path = "../opentelemetry-sdk", features = ["trace"] }
+opentelemetry-http = { version = "0.15", path = "../opentelemetry-http" }
```

Comment:
this should use 0.25, along with for semantic-convention ?

Reply:
Good catch!!
For next release onwards, I really need to automate this.